DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/RevitConverterToSpeckle.cs

```diff
@@ -49,4 +49,19 @@ public Base Convert(object target)
 
     return result;
   }
+
+  private IHostObjectToSpeckleConversion? GetConversionForObject(Type objectType)
+  {
+    if (objectType == typeof(object))
+    {
+      return null;
+    }
+
+    if (_toSpeckle.ResolveInstance(objectType.Name) is IHostObjectToSpeckleConversion conversion)
+    {
+      return conversion;
+    }
+
+    return GetConversionForObject(objectType.BaseType);
+  }
```

Comment:
ohhh, is this trying to resolve the converter including all parent type converters?

Neat! @BovineOx this is not a bad idea... could this be somehow moved into the default resolving logic or some new function of the converter (`ResolveInstance` and `ResolveInstanceWithInheritance`? name pending)

Reply:
Yeah it needs some thought, maybe add a POC: comment to consider making this more accessible as a pattern - make a CNX and add it here